ethereum/ratelimit/rate_limiter_test.go

```diff
@@ -19,9 +19,9 @@ var (
 	maxRequestsPer24HrsWithoutBuffer      = 301000
 	maxRequestsPer24hrs                   = maxRequestsPer24HrsWithoutBuffer - maxRequestsPer24HrsBuffer
 	maxRequestsPerSecond                  = 10.0
-	twentyFourHrs                         = (24 * 60 * 60 * 1000 * time.Millisecond)
+	twentyFourHrs                         = 24 * time.Hour
 	maxExpectedDelay                      = twentyFourHrs / time.Duration(maxRequestsPer24hrs)
-	minExpectedDelay                      = time.Duration(1000) / time.Duration(maxRequestsPerSecond) * time.Millisecond
+	minExpectedDelay                      = 1 * time.Second / time.Duration(maxRequestsPerSecond)
```

Comment:
@fabioberger I changed some things here in order to get the tests to pass. Let me know if this new version looks right to you.

`time.Duration(1000) / time.Duration(maxRequestsPerSecond) * time.Millisecond` in particular looks a little off to me. `time.Duration(1000)` is 1,000 nanoseconds, but I don't think that's what you intended.

Reply:
Thanks for re-writing this @albrow. The new version is much more readable and obviously correct.